openml/datasets/functions.py

```diff
@@ -368,7 +368,8 @@ def get_dataset(
     Parameters
     ----------
     dataset_id : int or str
-        Dataset ID of the dataset to download
+        Dataset ID of the dataset to download. It can be an integer or it can be a string
+        of the dataset name.
```

Comment:
```suggestion
        The ID or name of the dataset to download.
```
Since ID's are parsed to integer anyway if they're given as strings.

Reply:
@PGijsbers I can also do that, however, I tried to keep it consistent with https://github.com/openml/openml-python/blob/566943174bb7952721921c8d852112d127ae6901/openml/datasets/functions.py#L323-L325 at `get_datasets`

Reply:
I think the difference is fine, or should favor the simpler wording.